demos/demos.json

```
@@ -90,6 +90,15 @@
 					"description": "Showcases how Mavo works with [A-Frame](https://aframe.io/)—A web framework for building 3D/AR/VR experiences.",
 					"featured": false
 				},
+				{
+					"name": "Calculators",
+					"tag": [],
+					"id": "calculators",
+					"video": "/demos/calculators/video.mp4",
+					"url": "/demos/calculators",
+					"description": "Showcases how Mavo can gracefully handle calculations when two or more properties depend on each other. It also shows how Mavo can work with custom JavaScript functions as if they were Mavo built-in functions.",
```

```suggestion
					"description": "Showcases how mv-default can gracefully handle calculations with circular dependencies. It also showcases how Mavo functions can be extended by simply defining global JS functions (namely, a `gcd()` function).",
```

Also why do we need custom JS functions?!

---

It allows us to calculate elegantly the aspect ratio's “components” (`a` and `b` in `a:b`).
But we can try to avoid it in the next iteration.

---

Oh I posted that question? Thought I had deleted it after I looked up the function. No, it's fine.